redash/handlers/data_sources.py

```diff
@@ -52,6 +53,9 @@ def post(self, data_source_id):
         data_source.name = req['name']
         models.db.session.add(data_source)
 
+        # Refresh the stored schemas when a data source is updated
+        refresh_schemas.apply_async(queue="schemas")
```

Comment:
@emtwo Hey, after #3534 this and the other individual task calls should use the `SCHEMAS_REFRESH_QUEUE` setting now.

Reply:
thanks @jezdez, I've updated the PR.